crates/uv-fs/src/path.rs

```diff
@@ -83,6 +83,8 @@ pub fn normalize_url_path(path: &str) -> Cow<'_, str> {
 
 /// Normalize a path, removing things like `.` and `..`.
 ///
+/// Assumes that the path is already absolute.
```

Comment:
Why would we want to remove those? Should we add a debug assertion that they're not present at the beginning of the path?

Reply:
Why would we want to remove `..`?

Reply:
Is that what you're asking?